easybuild/easyconfigs/i/infercnvpy/infercnvpy-0.4.0-foss-2021b.eb

```
@@ -0,0 +1,47 @@
+easyblock = 'PythonBundle'
+
+name = 'infercnvpy'
+version = '0.4.0'
+
+homepage = 'https://github.com/icbi-lab/infercnvpy'
+description = "Infer copy number variation (CNV) from scRNA-seq data. Plays nicely with Scanpy."
+
+toolchain = {'name': 'foss', 'version': '2021b'}
+
+dependencies = [
+    ('Python', '3.9.6'),
+    ('SciPy-bundle', '2021.10'),
+    ('leidenalg', '0.8.8'),
+    ('scanpy', '1.9.1'),
+]
+
+use_pip = True
+
+# avoid hatcling requirement to install infercnvpy
+# (since installing it introduces conflicting version requirements with poetry included with Python)
+local_preinstallopts = """sed -i 's/^build-backend = .*/build-backend = "setuptools.build_meta"/g' pyproject.toml && """
+local_preinstallopts += """sed -i 's/^requires = .*/requires = ["setuptools"]/g' pyproject.toml && """
+local_preinstallopts += """sed -i 's/dynamic = \["version"\]/version = "%(version)s"/g' pyproject.toml && """
```

could just use one `sed` call with `-e` for each pattern
```suggestion
local_preinstallopts = """sed -i -e 's/^build-backend = .*/build-backend = "setuptools.build_meta"/g' """
local_preinstallopts += """ -e 's/^requires = .*/requires = ["setuptools"]/g' """
local_preinstallopts += """ -e 's/dynamic = \["version"\]/version = "%(version)s"/g' pyproject.toml && """
```

---

fixed in 6c51b07